modules/simul/src/main/SimulApi.scala

```diff
@@ -97,7 +97,7 @@ final class SimulApi(
             .filter(simul.variants.contains)
             .ifTrue(simul.nbAccepted < Game.maxPlayingRealtime)
             .so: variant =>
-              val perfType = PerfType(variant, chess.Speed.Rapid)
+              val perfType = PerfType(variant, chess.Speed(simul.clock.config.some))
```

Comment:
2 issues with this.

- it doesn't take into account the simul variant (neither does the current code, so the problem is already here)
- most simuls use classical clock, and most players don't have a classical rating. I reckon we should try using the simul speed, and if the user has no such rating, fallback to the next faster speed (so classical->rapid)

Reply:
I did notice the first issue when looking at this and had a short discussion about it in the [Discord](https://discord.com/channels/280713822073913354/352976626558042122/1169813531705757766).

I'll make more changes after tomorrow trying to fix these issues